main.go

```diff
@@ -52,23 +54,28 @@ func main() {
 	}
 
 	restServer := server.New(&cfg.DB, server.EnableAuth(cfg.Auth.Enabled))
+	mux := http.NewServeMux()
 	if cfg.Auth.Enabled {
 		log.Info("auth is enabled")
 		authHandler, err := auth.NewHandler(cfg.DB.URL, []byte(cfg.Auth.Secret))
 		if err != nil {
 			log.Fatal("initialize auth error ", err)
 		}
-		http.Handle("/auth/", authHandler)
+		mux.Handle("/auth/", authHandler)
 
 		middleware := auth.NewMiddleware([]byte(cfg.Auth.Secret))
-		http.Handle("/", middleware(restServer))
+		mux.Handle("/", middleware(restServer))
 	} else {
-		http.Handle("/", restServer)
+		mux.Handle("/", restServer)
 	}
 
+	// CORS Support
+	handler := cors.Default().Handler(mux)
```

Comment:
@YogiLiu do you need to make this a config item？
```
c := cors.New(cors.Options{
    AllowedOrigins: cfg.AllowedOrigins,
    AllowCredentials: true,
})
```

Reply:
@shellfly OK, it's done.